netty/src/main/java/io/atomix/catalyst/transport/NettyTls.java

```diff
@@ -15,71 +15,62 @@
  */
 package io.atomix.catalyst.transport;
 
-import javax.net.ssl.KeyManager;
-import javax.net.ssl.KeyManagerFactory;
-import javax.net.ssl.SSLContext;
-import javax.net.ssl.SSLEngine;
-import javax.net.ssl.SSLSocket;
-import javax.net.ssl.SSLSocketFactory;
-import javax.net.ssl.TrustManager;
-import javax.net.ssl.TrustManagerFactory;
 import io.atomix.catalyst.util.Assert;
-import java.io.FileInputStream;
-import java.security.KeyStore;
-import java.io.File;
-
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.net.ssl.*;
+import java.io.File;
+import java.io.FileInputStream;
+import java.security.KeyStore;
+
 /**
  * Netty TLS.
  *
  * @author <a href="http://github.com/electrical">Richard Pijnenburg</a>
  */
-
-public class NettyTls {
-
+final class NettyTls {
   private static final Logger LOGGER = LoggerFactory.getLogger(NettyTls.class);
-  private static boolean ts_use_ks = true;
-  private static KeyStore ts;
-  private static NettyProperties properties;
+  private NettyProperties properties;
 
   public NettyTls(NettyProperties properties) {
     this.properties = properties;
   }
 
-  public SSLEngine InitSSLEngine(Boolean client) throws Exception {
-
-    if (properties.sslTruststorePath() != null) {
-      ts_use_ks = false;
-    }
-
+  /**
+   * Initializes an SSL engine.
+   *
+   * @param client Indicates whether the engine is being initialized for a client.
+   * @return The initialized SSL engine.
+   */
+  public SSLEngine initSSLEngine(boolean client) throws Exception {
     // Load the keystore
-    KeyStore ks = loadKeystore(properties.sslKeystorePath(), properties.sslKeystorePassword());
+    KeyStore keyStore = loadKeystore(properties.sslKeyStorePath(), properties.sslKeyStorePassword());
```

Comment:
I gave more verbose names to the `ks` and `kmf` and those types of local variables simply because it's a convention in the Atomix projects to favor verbose names over acronyms.